examples/query-playground/src/QueryRepl.js

```diff
@@ -8,7 +8,8 @@ import { useTabs } from './hooks/useTabs'
 import { useExecuteQuery } from './hooks/useExecuteQuery'
 import styles from './QueryRepl.module.css'
 
-import './locales'
+/* eslint-disable-next-line import/no-unresolved */
+import './locales/index.js'
```

Comment:
*minor* Why change from `./locales`?  Do directory imports not work any more?

Reply:
It fails static analysis to use "bare filenames" with the eslint import plugin we added to cli-style.

Reply:
This is unfortunately going to require quite large changes in many places, I think....